File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int count = 0;
        int n = sc.nextInt();
        int[] arr = new int[n];
        int min = 1000000;
        int max = -1000000;
        for(int i=0;i<n;i++){
            arr[i] = sc.nextInt();
        }

        for(int j=0;j<arr.length;j++){
            if(arr[j]<min){
                min = arr[j];
            }
            if(arr[j]>max){
                max= arr[j];
            }
        }
        System.out.println(min+" "+max);

    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
n = int(input())
max = -1000000
min = 1000000
l = list(map(int, input().split()))
for i in range(n):
    if l[i]>max:
        max = l[i]
for i in range(n):
    if l[i]<min:
        min = l[i]
print(min,max)